src/core/utils/date.utils.ts

```diff
@@ -1,2 +1,8 @@
+import { DateTime } from 'luxon';
+
 export const API_MONTH_TO_FORMAT = 'yyyy-MM-01';
 export const API_MONTH_FROM_FORMAT = 'yyyy-MM-dd';
+
+// Expires date of cookies
+export const daysToExpire = new Date(DateTime.now().plus({ months: 10 }).toString());
+export const daysToDeleteCookie = new Date(DateTime.now().minus({ year: 2 }).toString());
```

Comment:
This should be generated dynamically with a function, otherwise, the value will remain the same till the server is restarted or a release is made because the values are in memory till that. Note that this doesn't happen or it is not noted on development because the server reloads the values on every change (hot reloading)

Also note, that if you're using `luxon` dates, you can do just `DateTime.utc().plus({ months: 10 }).toISO()` instead of creating a native date to parse it again.